operations/openstack/ephemeral-disk-vm-extensions.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Since this is the only _required_ ops-file for openstack, would it make sense to just call this `operations/openstack.yml`? That's currently our approach to other IaaS-specific ops-files.

Reply:
I'm not sure about that. I like the extra transparency of what this actually does. But sure, we can make this go in line with the other IaaS-specific ops files.